includes/class-img-converter.php

```
@@ -129,7 +129,7 @@ public function convert_image( string $source_image, string $format = 'webp', in
 		$max_bytes = apply_filters( 'wppo_filesize_limit_bytes', 20 * 1024 * 1024 );
 		if ( filesize( $source_image ) > $max_bytes ) {
 			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
-			error_log( sprintf( 'WPPO Error: Image filesize (%d bytes) exceeds limit (%d bytes) - %s', filesize( $source_image ), $max_bytes, $source_image ) );
+			error_log( sprintf( 'WPPO Error: Image filesize (%d bytes) exceeds limit (%d bytes) - %s', filesize( $source_image ), $max_bytes, str_replace( ABSPATH, '', $source_image ) ) );
```

PHPStan (2.2.5)

[warning] Constant ABSPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)